{"name": "test_backStage_authentication", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"http://10.10.128.152:10052/#/authority/mechanism ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ\")\n    @allure.testcase(\"http://10.10.128.152:10052/#/authority/mechanism\", \"ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ üëá\")\n    def test_backStage_authentication():\n        def_name = sys._getframe().f_code.co_name\n        logger.info(\"ÂºÄÂßãÊâßË°åËÑöÊú¨%s:\\n\", def_name)\n    \n        def_name = sys._getframe().f_code.co_name\n        logger.info(\"ÂºÄÂßãÊâßË°åËÑöÊú¨%s:\\n\", def_name)\n    \n        driver = webdriver.Chrome()\n        driver.maximize_window()\n        driver.get(url_back)\n        # Èì∂Ë°åÂØÜÁ†Å‰øÆÊîπ\n        # companyPassForget(driver, company_bank, 1)\n        # ÊãÖ‰øùÂÖ¨Âè∏ÂØÜÁ†Å‰øÆÊîπ\n>       companyPassForget(driver, company_Guarantee, 0)\n\ntest_flow\\test_Authentication\\test_backStage_authentication.py:82: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntest_flow\\test_Authentication\\test_login.py:118: in companyPassForget\n    waitUntilDisplay(driver, loginOn.txt_pwd_css.value)\ncommon\\BaseFunction.py:13: in waitUntilDisplay\n    WebDriverWait(driver, 30).until(EC.visibility_of_element_located((By.CSS_SELECTOR, ByCss)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"b487e3668c6c68a0e3e1491e446699d5\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x0000024A27FDDC10>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "description": "http://10.10.128.152:10052/#/authority/mechanism ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ", "attachments": [{"name": "log", "source": "970a2d0f-d589-489c-8a90-0894925308cb-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "e74d6d83-60f3-4c83-9380-2f083687b7e3-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a07346df-c956-467d-9ff8-41dafbc8a898-attachment.txt", "type": "text/plain"}], "start": 1599112199832, "stop": 1599112252767, "uuid": "6ead6626-7bf1-4fa2-b1a0-025ab0a20433", "historyId": "a55d1468f36f1b2cdddeddca0f290d7d", "testCaseId": "e04ecfcfa2b86e3258fad7c48a05b57e", "fullName": "test_flow.test_Authentication.test_backStage_authentication#test_backStage_authentication", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_backStage_authentication"}, {"name": "host", "value": "A02-03-00033"}, {"name": "thread", "value": "30148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_backStage_authentication"}], "links": [{"type": "test_case", "url": "http://10.10.128.152:10052/#/authority/mechanism", "name": "ÂêéÂè∞Ë¥¶Êà∑ËÆ§ËØÅ üëá"}]}